tst/CTA.WebForms2Blazor.Tests/FileConverterTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using NUnit.Framework;
+using CTA.WebForms2Blazor.Factories;
+using CTA.WebForms2Blazor.Services;
+using CTA.WebForms2Blazor.FileInformationModel;
+using System.Text;
+using System.Threading.Tasks;
+using CTA.WebForms2Blazor.FileConverters;
+
+namespace CTA.WebForms2Blazor.Tests
+{
+    [TestFixture]
+    class FileConverterTests
+    {
+        private const string TestFilesDirectoryPath = "TestingArea\\TestFiles";
+
+        //These are relative paths
+        private string _testProjectPath;
+        private string _testCodeFilePath;
+        private string _testConfigFilePath;
+        private string _testStaticFilePath;
+        private string _testViewFilePath;
+        private string _testProjectFilePath;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            _testCodeFilePath = Path.Combine(TestFilesDirectoryPath, "TestClassFile.cs");
+            _testConfigFilePath = Path.Combine(TestFilesDirectoryPath, "SampleConfigFile.config");
+            _testStaticFilePath = Path.Combine(TestFilesDirectoryPath, "SampleStaticFile.png");
+            _testViewFilePath = Path.Combine(TestFilesDirectoryPath, "SampleViewFile.aspx");
+            _testProjectFilePath = Path.Combine(TestFilesDirectoryPath, "SampleProjectFile.csproj");
+        }
+
+        [SetUp]
+        public void Setup()
+        {
+
+        }
+
+        [Test]
+        public async Task TestStaticFileConverter()
+        {
+            FileConverter fc = new StaticFileConverter(_testStaticFilePath);
+
+            IEnumerable<FileInformation> fileList = await fc.MigrateFileAsync();
+            FileInformation fi = fileList.First();
+            byte[] bytes = fi.FileBytes;
+
+            Assert.IsTrue(bytes.Length != 0);
+            Assert.IsTrue(fi.RelativePath.Equals("wwwroot\\TestingArea\\TestFiles\\SampleStaticFile.png"));
+        }
```

Comment:
A couple of suggestions for this test:

1. Rename test to `MigrateFileAsync_Moves_All_File_Bytes_To_wwwroot_Directory`. The ideal test name will include the function being tested, the condition of the test, and the expected outcome (e.g. `MyFunction_ThrowsException_With_Null_Parameter()`). Sometimes this is impossible or too wordy, so we can compromise based on context.
2. In the first assertion, we want to specify the exact number of bytes (e.g. what if the original file is 1 byte but the migrated file is 2 bytes?). Get the number of bytes of the original file and compare that to `fi.FileBytes.Length`.
3. Use `Path.Combine()` in the second `Assert` statement
4. Consider creating 1 test file per class, e.g. StaticFileConverterTests.cs. This will keep tests organized and prevent confusion if each converter class tests a shared interface with different implementations.